cmd/fyne/internal/mobile/gendex/gendex.go

```diff
@@ -59,12 +59,12 @@ func gendex() error {
 	if err := os.MkdirAll(tmpdir+"/work/org/golang/app", 0775); err != nil {
 		return err
 	}
-	javaFiles, err := filepath.Glob("../../../../vendor/github.com/fyne-io/mobile/app/*.java")
+	javaFiles, err := filepath.Glob("../../../../../internal/driver/mobile/app/*.java")
```

Comment:
Changing the path traversal depth does not seem right (internal is next to vendor) - are you sure?

Reply:
The generator is run manually, from the directory above.
This is required as that is where the generated file resides - it was probably put there to avoid a `main` package inside the `app` package.
The path traversal was correct, or at least it functions correctly before this change.

Reply:
Ah, you are right. Sorry for my naive thoughts. Indeed the `go:generate` is in a parent folder. I pushed a new commit which should fix the issue here. Thank you for catching this :)